app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -226,6 +226,29 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
         presenter.onViewReady()
     }
 
+    override fun onWindowFocusChanged(hasFocus: Boolean) {
```

Comment:
I'm not sure we need to do this here, we should be able to just set full screen in the onResume.

Reply:
Fullscreen is now set in onResume()